src/attached_probe.cpp

```diff
@@ -1155,7 +1163,7 @@ void AttachedProbe::attach_uprobe(int pid, bool safe_mode)
     return;
   }
 
-  if (!resolve_offset_uprobe(safe_mode))
+  if (!resolve_offset_uprobe(safe_mode, probe_.orig_name == probe_.name))
```

Comment:
So basically if multiple attachpoints are on a probe, we allow missing attachpoint with a warning. But if only one attachpoint is specified, it's an error.

I think this makes sense

Reply:
Yes, exactly. That's also how it works for kprobes.